site2/website/release-notes.md

```diff
@@ -1,6 +1,123 @@
 
 ## Apache
 
+### 2.3.0 &mdash; 2019-02-20 <a id="2.3.0"></a>
+
+#### General
+
+ * Support for schema definitions in the Pulsar [Python client library](https://pulsar.apache.org/docs/client-libraries-python/#schema)
+ * PIP-25: [Token based authentication](https://pulsar.apache.org/docs/security-token-client/) [#2888](https://github.com/apache/pulsar/pull/2888),
+   [#3067](https://github.com/apache/pulsar/pull/3067) and [#3089](https://github.com/apache/pulsar/pull/3089)
+ * Updated to [Apache BookKeeper 4.9.0](https://bookkeeper.apache.org/docs/4.9.0/overview/releaseNotes/)
+ * ZStandard compression codec [#3159](https://github.com/apache/pulsar/pull/3159). Note that
+   when a producer choose Zstd compression, a consumer will need to be at least at version 2.3.0
+   to be able to correctly receive the messages.
+ * Support for Java 11 [#3006](https://github.com/apache/pulsar/pull/3006)
+ * `AUTO_PRODUCE` Schema type to allow to publish serialized data and validate it against the
```

Comment:
I think `AUTO_CONSUME` and generic record is also in 2.3.0

Reply:
I think it was there in 2.2 though it was called "AUTO" since it was used in the JDBC sink

Reply:
okay. I remember there is one more schema was introduced for debezium by @jiazhai . hardly remember the name. was it KeyValue schema?

Reply:
Added keyvalue schema